workflows/python/sdk/order-processor/workflow_activities.py

```diff
@@ -0,0 +1,101 @@
+
+from datetime import timedelta
+import logging
+import json
+from dapr.ext.workflow import DaprWorkflowContext, WorkflowActivityContext
+# , when_any
+from model import InventoryItem, Notification, InventoryRequest, OrderPayload, OrderResult, PaymentRequest, InventoryResult
+from dapr.clients import DaprClient
+from util import get_address
+
+store_name = "statestore-actors"
+
+address = get_address()
+
+def order_processing_workflow(ctx: DaprWorkflowContext, orderPayload: OrderPayload):
+    logging.basicConfig(level=logging.INFO)
```

Comment:
We shouldn't be configuring logging inside the workflow function since it will get re-executed multiple times. I suggest doing it outside the workflow function, perhaps on the file scope.

Reply:
Done